examples/SteadyStateSimulator/IllustrativeExamples/description/IllustrativeExamples.tex

```diff
@@ -0,0 +1,208 @@
+}
+
+\begin{document}
+\chapter*{Illustrative examples for the steady state simulator}
+
+This document presents the results of several simple test cases to illustrate the possibilities of the steady state simulator.
+
+\section*{Phase Shifters in parallel (PhaseShifters)}
+
+The following system is simulated. It is made of a generator (GeneratorPVSignalN), two phase shifter transformers, a transformer, five lines and a PQ load (LoadPQ).\\
+
+\begin{figure}[H]
+  \begin{center}
+  \includegraphics[width=0.8\textwidth]{PhaseShifters/PhaseShifters}
+  \end{center}
+  \caption{Illustration of the studied system}
+\end{figure}
+
+The line is disconnected at t = 30s.\\
+Two simulations are done :
+\begin{itemize}
+\item the two phase shifters have the following time constants t1st = 40s/tNext = 20s;
+\item one phase shifter have the following time constants t1st = 20s/tNext = 10s, the other one t1st = 40s/tNext = 20s.
+\end{itemize}
+
+The currents in both phase shifters are compared in both cases. One can observe that depending on the time constants of the phase shifter, the steady state after the event is different.\\
+\\
+This cannot be captured by a fully static simulator, in which all the phase shifters are activated at the same time with an external loop.
+
+\begin{figure}[H]
+  \begin{tikzpicture}
+    \begin{axis}[xmin = 0, ymin = 0.24, ymax = 0.34, height = 2.3in, legend pos=north east]
+        \addplot[color=red!50,line width=0.03in]
+        table[x=time,y=PhaseShifter_5_6_phaseShifter_iMonitored]
+        {PhaseShifters/curves2010.csv};
+        \addplot[color=blue!50,line width=0.03in]
+        table[x=time,y=PhaseShifter_5_6_phaseShifter_iMonitored]
+        {PhaseShifters/curves4020.csv};
+        \legend{20/10, 40/20}
+        \end{axis}
+  \end{tikzpicture}
+  \caption{Current in the first phase shifter [p.u]}
+\end{figure}
+
+\begin{figure}[H]
+  \begin{tikzpicture}
+    \begin{axis}[xmin = 0, ymin = 0.24, ymax = 0.34, height = 2.3in, legend pos=north east]
+        \addplot[color=red!50,line width=0.03in]
+        table[x=time,y=PhaseShifter_5_7_phaseShifter_iMonitored]
+        {PhaseShifters/curves2010.csv};
+        \addplot[color=blue!50,line width=0.03in]
+        table[x=time,y=PhaseShifter_5_7_phaseShifter_iMonitored]
+        {PhaseShifters/curves4020.csv};
+        \legend{20/10, 40/20}
+        \end{axis}
+  \end{tikzpicture}
+  \caption{Current in the second phase shifter [p.u]}
+\end{figure}
+
+\section*{Simplified HVDC link with AC emulation and phase shifter (HVDC\_PhaseShifter)}
+
+The following system is simulated. It is made of a generator (GeneratorPVSignalN), a phase shifter transformer, four lines, two transformers, a simplified HVDC link with an AC emulation (SimplifiedHVDCACEmulation) and a PQ load (LoadPQ).\\
+
+\begin{figure}[H]
+  \begin{center}
+  \includegraphics[width=0.8\textwidth]{HVDC_PhaseShifter/HVDC_PhaseShifter}
+  \end{center}
+  \caption{Illustration of the studied system}
```

Comment:
I have a problem with the generation of this image

Reply:
It was a problem. Thank you !